ownership-chain/precompile/laos-evolution/contracts/LaosEvolution.sol

```diff
@@ -26,6 +26,14 @@ interface LaosEvolution {
         uint256 tokenId
     );
 
+    /// @notice Emitted when a token metadata is updated
+    /// @param tokenId the resulting id of the newly minted token
+    /// @param collectionId the id of the collection
+    event MetadataUpdate(
```

Comment:
Token metadata needs to be part of the event otherwise otherwise it will be possible retrive it only by archive nodes.